.storybook/locale.ts

```diff
@@ -233,7 +232,7 @@ const locale: Record<string, string> = {
   siteBannerConnectedText: 'This creator has not yet signed up to receive contributions from Brave users. Your browser will keep trying to contribute until they verify, or until 90 days have passed.',
   sites: 'sites',
   tellOthers: 'Tell others about your tip.',
-  supportedSites: 'Supported Sites',
+  supportedSites: 'Sites viewed',
```

Comment:
If this is a title, would expect 'Sites Viewed'

Reply:
it's not a title, it's a tab